package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "lighthouses",
-  "version": "0.0.1",
+  "version": "2.0.0",
```

Comment:
For now, let's keep it in v0 because it's not a published package. Perhaps we could do `0.0.2`?
```suggestion
  "version": "0.0.2",
```

Reply:
I like it!

Reply:
For more context, my thinking is from the [semver spec](https://semver.org/#spec-item-4).